backend/TaskTree/Models/Requests/CreateProjectRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using TaskTree.Models.Responses;
+
+namespace TaskTree.Models.Requests
+{
+    public class CreateProjectRequest
+    {
+        public string Name { get; set; } = null!;
+        public string? Description { get; set; }
+        public TaskResponse? Root { get; set; }
```

Comment:
there wont be a task passed in on creation. if we decide that creating a project creates a root task this will be done in this controller and not passed in.